Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -347,7 +347,7 @@ public class HTTPServer: Server {
         }
 
         let bootstrap = ServerBootstrap(group: eventLoopGroup)
-            .serverChannelOption(ChannelOptions.backlog, value: BacklogOption.Value(self.maxPendingConnections))
+            .serverChannelOption(ChannelOptions.backlog, value: ChannelOptions.Types.BacklogOption.Value(self.maxPendingConnections))
```

Comment:
use `ChannelOptions.backlog`, that works in all NIO versions. The `Types` are more or less internal